resources/charts/nethealth/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+appVersion: 0.1.0
+description: Nethealth collects information about the overlay network.
+home: https://github.com/gravitational/monitoring-app
+keywords:
+- nethealth
+kubeVersion: '>=1.15.0-0'
```

Comment:
can you link where this version format is described?

Reply:
Thanks - that helps! I was more curious about the `1.15.0-0` notation (the `-0` suffix) - could not find a justification for it in the docs.